keyvault/spring-cloud-azure-starter-keyvault-secrets/single-property-source/terraform/variables.tf

```diff
@@ -1,17 +1,17 @@
 variable "application_name" {
   type        = string
-  description = "The name of your application"
+  description = "The name of your application."
   default     = "keyvault"
 }
 
 variable "location" {
   type        = string
-  description = "The Azure region where all resources in this example should be created"
+  description = "The Azure region where all resources in this example should be created."
   default     = "eastus"
 }
 
 variable "sample_tag_value" {
   type        = string
-  description = "The value of spring-cloud-azure-sample tag"
+  description = "The value of spring-cloud-azure-sample tag."
```

Comment:
How about delete this variable and write it directly like this:

```
resource "azurerm_resource_group" "main" {
  name     = azurecaf_name.resource_group.result
  location = var.location

  tags = {
    "terraform"                 = "true"
    "application-name"          = var.application_name
    "spring-cloud-azure-sample" = "true"
  }
}
```

Reply:
Same in other variables.tf files.

Reply:
it can't because of syntax

Reply:
Why `"terraform"                 = "true"` is OK and `"spring-cloud-azure-sample" = "true"` has syntax error?

Reply:
sorry, misunderstood your notes, 

I don't think it's in this scope.

It's in the template, not in this scope, we can talk about it in another issue.